rootfs/scheduler/__init__.py

```diff
@@ -936,62 +936,6 @@ def delete_rc(self, namespace, name):
 
         return response
 
-    def _healthcheck(self, namespace, container, routable=False, path='/', port=5000,
```

Comment:
I find it a little dangerous when we are not keeping the healthcheck logic inside the scheduler but it is rather now outside in serializer class... Thoughts?